packages/main/src/Avatar.ts

```diff
@@ -53,12 +54,9 @@ import "@ui5/webcomponents-icons/dist/alert.js";
  * <code>import "@ui5/webcomponents/dist/Avatar.js";</code>
  *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.main.Avatar
- * @extends sap.ui.webc.base.UI5Element
- * @tagname ui5-avatar
+ * @extends UI5Element
  * @since 1.0.0-rc.6
```

Comment:
- Remove type tag
- Remove readonly and type tags from getter. Add explicit return type